automation/check-patch.e2e-nmstate-functests.sh

```diff
@@ -40,12 +40,13 @@ main() {
     cd ${TMP_COMPONENT_PATH}
 
     make test-e2e-handler \
-        E2E_TEST_TIMEOUT=${TIMEOUT} \
-        E2E_TEST_ARGS="-ginkgo.noColor --junit-output=$ARTIFACTS/junit.functest.xml" \
-        HANDLER_NAMESPACE=$NAMESPACE \
+        E2E_TEST_TIMEOUT=$TIMEOUT \
+        e2e_test_args="-noColor" \
+        E2E_TEST_SUITE_ARGS="--operator-namespace $NAMESPACE --junit-output=$ARTIFACTS/junit.functest.xml" \
```

Comment:
This change seems unrelated. Please keep it in a separate commit.

Reply:
Done